src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -4870,6 +4870,28 @@ public void testCreateService() throws Exception {
   }
 
   @Test
+  public void testCreateServiceWithWarnings() throws Exception {
```

Comment:
@fdonze I see the [docs here](https://docs.docker.com/engine/api/v1.25/#operation/ServiceCreate) are as you said. I wanted to create a PR for them. Where's the source code for the docs? Doesn't seem to be [here](https://github.com/moby/moby/tree/master/docs/api).